### DISCUSSION TITLE
是否可以为侵蚀一练级添加使用塞壬探测装置的功能

### DISCUSSION THREAD

**Discussion by LouiseShariarc** [LmeSzinc/AzurLaneAutoScript#2899] [General]
接近月末时经常出现因黄币不足而中断侵蚀一练级的事件，是否可以为侵蚀一练级添加使用塞壬探测装置的功能以将溢出紫币兑换为黄币

**Comment by mikage3miya** [comment]
建议考虑调整大世界通用设置，关闭紫币的购买

**Reply by AkiraChisaka** [reply]
感谢建议！

顺便问一下，大世界里紫币有什么用处么？

如果港口商店里东西都买光了，是不是就不用买紫币了？

**Comment by AkiraChisaka** [comment]
嗯，我也有同样的问题。黄币严重的不足，侵蚀一每天基本只能刷几个小时。
